R/whatWQPdata2.R

```diff
@@ -0,0 +1,106 @@
+#' @return A data frame with at least the following columns:
+#' \tabular{lll}{ 
+#' Name \tab Type \tab Description \cr
+#'  "Provider" \tab character \tab Providing database.  \cr            
+#'  "MonitoringLocationIdentifier" \tab character \tab	A designator used to 
+#' describe the unique name, number, or code assigned to identify 
+#' the monitoring location.\cr
+#'  "YearSummarized" \tab numeric \tab The year of the summary \cr
+#'  "CharacteristicType" \tab character \tab CharacteristicType  \cr  
+#'  "CharacteristicName" \tab character \tab	The object, property, or substance 
+#' which is evaluated or enumerated by either a direct field measurement,
+#' a direct field observation, or by laboratory analysis of material
+#' collected in the field.\cr 
+#'  "ActivityCount" \tab numeric \tab The number of times the location was sampled \cr
+#'  "ResultCount" \tab numeric \tab The number of individual data results. \cr
+#'  "LastResultSubmittedDate" \tab Date \tab Date when data was last submitted. \cr
+#'  "OrganizationIdentifier" \tab character \tab  A designator used to uniquely 
+#' identify a unique business establishment within a context.\cr
+#'  "OrganizationFormalName" \tab character \tab  The legal designator
+#' (i.e. formal name) of an organization.\cr
+#'  "MonitoringLocationName \tab character \tab MonitoringLocationName \cr 
+#'  "MonitoringLocationTypeName" \tab character \tab MonitoringLocationTypeName \cr
+#'  "ResolvedMonitoringLocationTypeName" \tab character \tab  \cr
+#'  "HUCEightDigitCode" \tab character \tab 8-digit HUC id. \cr
+#'  "MonitoringLocationUrl" \tab character \tab URL to monitoring location. \cr
+#'  "CountyName" \tab character \tab County of sampling location. \cr
+#'  "StateName" \tab character \tab State of sampling location. \cr  
+#'  "MonitoringLocationLatitude"  \tab numeric \tab latitude of sampling 
+#'  location. \cr
+#'  "MonitoringLocationLongitude" \tab numeric \tab longitude of sampling 
+#'  location. \cr
+#' }
+#' @export
+#' @examplesIf is_dataRetrieval_user()
+#' \donttest{
+#' dane_county_data <- whatWQPdata2(countycode = "US:55:025",
+#'                         summaryYears = 5,
+#'                         siteType = "Stream")
+#' 
+#' 
+#' lake_sites <- whatWQPdata2(siteType = "Lake, Reservoir, Impoundment",
+#'                          CharacteristicName = "Temperature, water",
+#'                          countycode = "US:55:025")
+#' # Compare with the the results of whatWQPdata:                        
+#' lake_sites2 <- whatWQPdata(siteType = "Lake, Reservoir, Impoundment",
+#'                          CharacteristicName = "Temperature, water",
+#'                          countycode = "US:55:025")
+#'                          
+#' }
+whatWQPdata2 <- function(...){
+  
+  values <- readWQPdots(..., zip = "no")
+  
+  values <- values$values
+
+  if("service" %in% names(values)){
+    values <- values[!(names(values) %in% "service")]
+  }
```

Comment:
Cool 🤯I think this is a relatively new feature of Codecov. @lstanish-usgs if you want to make any line-by-line comments on the code, you can do it by clicking the code line. Maybe you knew that, but if not - it's a great feature (same works in code.usgs.gov)